### PR TITLE
Custom Drawer

### LINKED ISSUE #26: Detailed implementation of Drawer
## Kind (Required)

- Improvement

 ## Overview (Required)

- This is Rough Issue
- Drawer has some items
- It would be best if there was a button to switch to a dark theme

 ## Links	

-

### PR DESCRIPTION
**Pull request by b4tchkn**
## TL;DR
Customed drawer layout to add Settings item and dark theme switch

- close #26 
## Why we need this change? (required)
To access settings and dark theme

## What does this change? (required)
add Settings item and dark theme switch

## What is the value of this and can you measure success? (required)
please check below gif

## How it was implemented
I added constraintlayout in NavigationView.

## Where struggled
First, I had no idea to implement custom drawer layout but it like normal layout (ex. like activity, fragment

## Reference links
- [【Android/Kotlin】いまさら聞けない？NavigationDrawerのメニューをカスタマイズする方法](https://marketable-skill.biz/?p=966)
- [トグルボタンの実装](https://qiita.com/nein37/items/850824d2b35663111ddb)
## Screenshot

Before | After
:--: | :--:
 | 

## Notes

- If you are OK, please approve this PR.
- If anyone approve this PR, I'll merge this PR myself.

### CODE REVIEW

**Comment by ho2ri2s** [comment]
ほほう、menu指定だと単純なリストっぽいのしかできないけど、NavigationViewを親にネストして自由にViewがかけるのか便利やな👍
commitも見やすいしPRの粒度も完璧！

**Review by ho2ri2s — APPROVED** [review]
ほほう、menu指定だと単純なリストっぽいのしかできないけど、NavigationViewを親にネストして自由にViewがかけるのか便利やな👍
commitも見やすいしPRの粒度も完璧！
